test/api/routing.test.js

```diff
@@ -82,3 +83,67 @@ describe('Routing', () => {
     });
   });
 });
+
+describe('Matrix', () => {
```

Comment:
looking at our other test files with multiple endpoints (geocoding, search), it seems like there's one top-level `describe` block, with each API nested as it's own `context`, see the following examples from search:
1. [searchPlaces](https://github.com/radarlabs/radar-sdk-js/blob/master/test/api/search.test.js#L44)
2. [searchGeofences](https://github.com/radarlabs/radar-sdk-js/blob/master/test/api/search.test.js#L83)
3. [autocomplete](https://github.com/radarlabs/radar-sdk-js/blob/master/test/api/search.test.js#L122)

could we follow the same pattern here and nest a `getMatrixDistances` context within the top-level `Routing` block?

apologies about being super nitpicky here, it's just easier for a new dev to reason about the tests if they follow the same structure

Reply:
nope, this makes sense! thanks @jsani-radar - updated